test/apis/stacks.test.js

```diff
@@ -36,7 +35,7 @@ test('stacks.get', t => {
   td.verify(requestStub(td.matchers.contains(expectedArgs), td.matchers.anything()))
 })
 
-test('stacks.create', t => {
+test('stacks.createOld', t => {
```

Comment:
I like this. Maybe call it `stacks.create (version <=0.26)` as there's no such function.